leakcanary-android-release/src/main/java/leakcanary/GoodAndroidVersionCondition.kt

```diff
@@ -0,0 +1,25 @@
+package leakcanary
+
+import android.os.Build
+import leakcanary.HeapAnalysisCondition.Result.StartAnalysis
+import leakcanary.HeapAnalysisCondition.Result.StopAnalysis
+
+class GoodAndroidVersionCondition : HeapAnalysisCondition() {
+  private val cachedResult: Result by lazy {
+    val sdkInt = Build.VERSION.SDK_INT
+    if (// findObjectById() sometimes failing. See #1759
+        sdkInt != 23 &&
+        // findObjectById() sometimes failing. See #1759
+        sdkInt != 25 &&
+        // Android 11 seem to sometimes have super slow heap dumps.
+        // See https://issuetracker.google.com/issues/168634429
+        sdkInt < 30
```

Comment:
Suggestion: have you considered adding `sdk > 21`? On older Android phones it much be too much of a pressure to run analysis in background. Or, maybe, make it configurable?

Reply:
It's already configurable, anyone can create and use their own conditions :) . I'm hoping some day I can fix these / reenable them so I don't want to provide that as a parameter. This identifies the versions *we* know might not work.

As for memory / pressure, we have other conditions for that, I don't see why pre 21 would be super different.